.drone.yml

```diff
@@ -143,9 +147,15 @@ trigger:
   - pull_request
   paths:
     exclude:
-    - '*.md'
     - docs/**
-    - latest.json
+    - '*.md'
+    include:
+    - pkg/**
+    - packaging/*
```

Comment:
should this be a double-star pattern to match all deeply nested files?

Reply:
true, thanks! fixed.